src/test/java/com/orbitz/consul/util/HttpTest.java

```diff
@@ -0,0 +1,306 @@
+package com.orbitz.consul.util;
```

Comment:
OPT: break lines between Arrange/Act/Assert sections in tests when they are more than a couple of lines

Reply:
Done.
Please tell me if it meets your expectations.

Reply:
👌